packages/config/src/config.js

```diff
@@ -92,7 +92,7 @@ module.exports = ({
                                  * Context class is equal to app name and that class ass added to root element via the chrome-render-loader.
                                  */
                                 if (relativePath.match(/^src/)) {
-                                    return `.${appName}${sassPrefix ? `, .${sassPrefix}` : ''}{\n${content}\n}`;
+                                    return `${sassPrefix || `.${appName}`}{\n${content}\n}`;
```

Comment:
I think you are missing the `.` before the prefix. Unless we expect the user to input prefix with the dot.

Reply:
Yup, we expect the user to do that. The prefix can be any sass prefix for instance `#something .oneClass, .rbac`

Reply:
:+1: